src/Processors/Sources/RemoteSource.h

```diff
@@ -26,6 +27,8 @@ class RemoteSource : public ISource
     Status prepare() override;
     String getName() const override { return "Remote"; }
 
+    void connectToScheduler(ResizeProcessor & scheduler);
```

Comment:
Let's maybe replace resize_processor to `InputPort &` ? And remove `getFreeInputPortIfAny`